LAB3.html

```diff
@@ -0,0 +1,195 @@
bbb2fBggWccsopTJ06laobbriB6667jqonnniCGTNmcO2117KvF198kZNPPplTTjmFb3/723R0dLB06VIuvfRSjjnmGG655Rb298Mf/hBJXHXVVXR0dLB48WImT57Ml7/8Za677jpmzJjBjBkzqGpvb2fGjBnMmDGDb3/725x66qksXbqUJUuWcNNNN1FRKpU488wz+cxnPsPChQvp6Oigo6ODm266iWnTpnH88cfz4osvUrVkyRJmzJjBjBkzWLlyJZL46le/SkdHB4sXL+akk07iv/7X/8oNN9zAhAkT+Id/+Ac6OjpYuHAhn/zkJ7n//vuJiHivKYiIiLfl+eef5+abb+bmm2/m5ptv5m//9m9pa2vjjDPOoGLIkCF84xvfoGrhwoXcc889VBxxxBG0trZy1llnccghh1DR3d3NVVddxb4ef/xxpk2bxt69e6n42Mc+RmtrK8cddxz/8A//QG9vL++0sWPHUvXMM89QtXDhQu655x4qjjjiCFpbWznrrLM45JBDqOju7uaqq67irVq4cCH33HMPFUcccQStra2cddZZHHLIIVR0d3dz1VVX8Xb80z/9E7NmzWLWrFnMmjWLWbNmMWvWLGbNmkV/fz9vx/nnn8/mzZupOPvss/mbv/kbLrjgAg477DAqli9fzg033MBbtWfPHhobG+nq6qKipqaGP/7jP+YP/uAPqHjjjTe44IILuO2226jauHEj06dPp7e3l4qPf/zjtLa2ctxxx3HdddexZs0afplvfvOb7N27l6ovfelLVCxcuJB77rmHiiOOOILW1lbOOussDjnkECq6u7u56qqrOJBzzjmHuro6zj77bMaPH0/VrFmzuPTSSznyyCNpbW3lIx/5CBW9vb381V/9FRER7zUFERHxtmzatIkLL7yQCy+8kAsvvJBrr72WW2+9lTfeeIOamhruvPNOTj75ZKp27tzJ+PHjqaur47HHHmPZsmXceeed3HXXXVRt2bKFfV133XVUnXHGGfzzP/8zy5Yt4+c//zl/9md/xm/D8OHDqdqzZw9VO3fuZPz48dTV1fHYY4+xbNky7rzzTu666y6qtmzZQsWyZcvo7Oyk6vTTT6dcLvOP//iPVO3cuZPx48dTV1fHY489xrJly7jzzju56667qNqyZQtvx+bNm7n++uu5/vrruf7667n++uu5/vrruf766xkYGOCtKpVKrFmzhorJkydzxx13cM0113DzzTezevVqpkyZwgUXXMDHP/5xKpYtW0ZnZydVp59+OuVymX/8x3+kauHChXR3d1Px0Y9+lH/5l39h3bp1vPjii1x33XVUzZ49m9dee42Ka6+9loGBASrOOOMMfv7zn7Ns2TJ+/vOfc9ZZZ/HGG2/wywwMDLB06VJ6e3u57777OPXUU6nYuXMn48ePp66ujscee4xly5Zx5513ctddd1G1ZcsWDmT06NE8+eST3HHHHaxfv56DDz6YioGBASZOnMg//dM/sWzZMrq6uqiyTV9fHxER7yUFERHxjhkYGOALX/gC3/zmN6maP38+GzduZM+ePRx11FFU9Pf38+qrr1Jlm33dddddVF1zzTUURUHVvHnz+G04+OCDOZD58+ezceNG9uzZw1FHHUVFf38/r776KlW2eavmz5/Pxo0b2bNnD0cddRQV/f39vPrqq1TZ5t1QFAVDhgyh4sEHH+Tkk0/m61//OmvXruVTn/oUq1at4uabb+bUU0/lrVq6dClVN954I2PGjKHqsssu45RTTqFi165d/OhHP6Linnvuoerqq6+mKAqqrrnmGn6VU045hXPPPZehQ4dy2mmnUTV//nw2btzInj17OOqoo6jo7+/n1Vdfpco2B/KlL32Jww47jIr6+nomT55M1V/+5V/S0NBAxVFHHcXw4cOp6uvrIyLivaQgIiLelilTprBr1y527drFrl272Lx5M6tXr+bKK6+kYmBggP/8n/8zq1atoqqvr48VK1bQ1tbGpEmTqK+v55xzzqGqp6eHqv7+ft544w0qampq+MQnPsG+JkyYQE1NDe+0l19+mara2lr21dfXx4oVK2hra2PSpEnU19dzzjnnUNXT08Pb0dfXx4oVK2hra2PSpEnU19dzzjnnUNXT08PbMWXKFHbt2sWuXbvYtWsXu3btYteuXezatYu6ujrejlmzZlHV1dXFFVdcwSmnnMJBBx3EOeecw0MPPcRb1dfXxzPPPENFTU0NTU1N7K+pqYmqdevW0d/fzxtvvEHVhAkT2NfYsWMZMmQIv8y4ceN4M319faxYsYK2tjYmTZpEfX0955xzDlU9PT0cyCc+8Qn2VRQFVR/5yEfY17Bhw4iIeK8qiIiIt6W2tpaGhgYaGhpoaGhgzJgxNDc387WvfY1p06ZR9f3vf5+KdevWMWbMGFpbW7n11ltZv349Rx11FNOmTaNq5MiRVJVKJaqKouBAamtreaft2LGDqpEjR1K1bt06xowZQ2trK7feeivr16/nqKOOYtq0aVSNHDmSt2rdunWMGTOG1tZWbr31VtavX89RRx3FtGnTqBo5ciRvR21tLQ0NDTQ0NNDQ0EBDQwMNDQ00NDTwdn3zm9/kyiuv5IgjjmBfvb29rFixgsmTJ/P1r3+dt6Kvr4+q2tpaiqJgfx/+8Iep6u/vp1Qq8avU1tbyy/z+7/8+B7Ju3TrGjBlDa2srt956K+vXr+eoo45i2rRpVI0cOZIDKYqCN1MUBRER7xcFERHxjvnEJz5B1Z49eyiVSrS2trJt2zYqFixYwLZt23j22WdZvHgxVSNHjqRq6NCh1NTUULF37152797Nvnp6eujt7eWd1NPTw6pVq6g6/fTTqSiVSrS2trJt2zYqFixYwLZt23j22WdZvHgxVSNHjuStKJVKtLa2sm3bNioWLFjAtm3bePbZZ1m8eDFVI0eO5N30ta99jZdeeomf/vSnXHHFFfzJn/wJQ4YMoerv//7v6e/v51c56KCDqKmpoaK3t5cdO3awv0cffZ
```

Comment:
i thought if we have a sigma that is under five sigma there is no significance. can you explain it to me. thank you

Reply:
I interpret "significance" here to mean the value we calculated. The sigma values (5 sigma, 3 sigma, etc.) we've been comparing to are the common thresholds set by the scientific community used for claiming that we have evidence of a discovery.

Reply:
I interpret "significance" here to mean the value we calculated. The sigma values (5 sigma, 3 sigma, etc.) we've been comparing to are the common thresholds set by the scientific community used for claiming that we have evidence of a discovery.